.github/workflows/release-binaries-save-stage/action.yml

```diff
@@ -0,0 +1,33 @@
+name: Save Stage
```

Comment:
Can we get a comment somewhere in here about what this exactly is used for? Something like the description flag in the `setup-stage` action below?